### ISSUE TITLE
Contact form does not reach the bottom of the page 

### ISSUE THREAD

**Issue by RQ16** [facn2/week1-Ba6e5-Developers#40]
If you scroll down to the bottom of the page, the form has white space underneath it

**Comment by RQ16** [comment]
It seems to be because the div with the "form-content" class in CSS has top : 10%

**Comment by matthewdking** [comment]
@RQ16 I'm sorry the form looks good to me! Can you clarify the issue more? Maybe send me a screenshot of the problem?

**Comment by RQ16** [comment]
Hi, @matthewdking! I attached screenshots of before and after the form was fixed. Make sure you click on the images to see the contact section and white space below it. 
Before:
![imageedit_2_8213534202](https://user-images.githubusercontent.com/24395716/27963462-d38d5da8-633d-11e7-98a7-005eb5f4db75.png)
After:
![rsz_form_fixed](https://user-images.githubusercontent.com/24395716/27963467-d5ff7bde-633d-11e7-8dcc-586b09f2fcff.png)
